tfe/resource_tfe_team.go

```diff
@@ -22,14 +23,38 @@ func resourceTFETeam() *schema.Resource {
 			"name": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
 			},
 
 			"organization": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
+
+			"organization_access": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				MinItems: 1,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"manage_policies": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+
+						"manage_workspaces": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+
+						"manage_vcs_settings": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
```

Comment:
I don't this these should be required, right? They should be optional and I would give them a default that is inline with the defaults using in the UI when creating a new group (so I think the default should be `false`).

Reply:
See my comment above. Still think so? I can change if you like, but that was my thinking.

Reply:
I guess I don't care either way, defaulting is fine; I'll change it.